spec/fixtures/files/prometheus2.debian

```diff
@@ -7,7 +7,7 @@
 # Default-Stop:      S 0 1 6
 # Short-Description: Prometheus monitoring system
 # Description:       The Prometheus monitoring system and time series database. https://prometheus.io/
-#                    
+#
```

Comment:
is this file even used anymore or can we delete it?

Reply:
I wanted to clean up all files in `spec/fixtures/files` in a future PR